build-container-image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Are we certain, that after building the docker image, everything works? Don't we need some kind of validation that the image was actually build?